### PR TITLE
[fix](fe) Fix drop table write too many times editlog

### PR DESCRIPTION
**Pull request by swjtu-zhanglei**

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java [hunk 1]
```diff
@@ -1125,7 +1125,7 @@ public static void loadJournal(Env env, Long logId, JournalEntity journal) {
                     break;
                 }
                 case OperationType.OP_DELETE_TABLE_STATS: {
-                    env.getAnalysisManager().replayTableStatsDeletion((TableStatsDeletionLog) journal.getData());
+                    LOG.info("replay delete table stat: {}", ((TableStatsDeletionLog) journal.getData()).id);
```
Comment (w41ter) [inline fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java:1128]: Without replay table stats deletion, how to clear a follower's table stats?

**Comment by SWJTU-ZhangLei** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java (hunk 1 above)
Comment (SWJTU-ZhangLei) [reply fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java]: > Without replay table stats deletion, how to clear a follower's table stats?

Thanks for you comment, i hava updated the code.

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by w41ter — APPROVED** [review]

**Pull request merged**